src/Mvp/Crud/CrudView.php

```diff
@@ -30,7 +30,19 @@
 
 class CrudView extends HtmlView
 {
-    public $newModel = true;
+    protected $title;
+    protected $newModel = true;
+
+    public function __construct()
+    {
+        $this->attachEventHandler("SetTitle", function ($title) {
```

Comment:
I thought the same while doing it, the only issue is that changing it to be the other way around will have more knock-on effects in existing projects where the view classes are referring to their title property and just expecting it to have its value set already. If you'd rather we bite that bullet for consistency of the approach, I'll start a confluence doc for things to be aware of when migrating from core to rhubarb.